lib/backend/registrybackend/security/security.go

```diff
@@ -48,14 +48,15 @@ type Config struct {
 	TLS                    httputil.TLSConfig `yaml:"tls"`
 	BasicAuth              *types.AuthConfig  `yaml:"basic"`
 	RemoteCredentialsStore string             `yaml:"credsStore"`
+	EnableHTTPFallback     bool               `yaml:"enableHTTPFallback"`
```

Comment:
We use underscore instead of camel case in most places.
Not sure we used camel case here to begin with...I will switch it to camel case in another PR, just FYI.